mantle/vendor/github.com/coreos/coreos-assembler-schema/cosa/build.go

```diff
@@ -178,7 +178,7 @@ func (build *Build) GetArtifact(artifact string) (*Artifact, error) {
 	if ok && r.Path != "" {
 		return r, nil
 	}
-	return nil, errors.New("artifact not defined")
+	return nil, errors.New("artifact " + artifact + " not defined")
```

Comment:
if you update this file you also have to update https://github.com/coreos/coreos-assembler/blob/main/schema/cosa/build.go